README.md

```diff
@@ -30,6 +30,9 @@ This repository contains OpenTofu Core, which includes the command line interfac
 
 - To submit bug reports or enhancement requests, refer to the [contributing guide](CONTRIBUTING.md) as well.
 
+## Reporting security vulnerabilities
+If you've found a vulnerability or a potential vulnerability in OpenTofu please let us know at [security@opentofu.org](mailto:security@opentofu.org). We'll send a confirmation email to acknowledge your report, and we'll send an additional email when we've identified the issue positively or negatively.
```

Comment:
Should this not link to `SECURITY.md` instead of repeating _some_ of that content here?